### PR TITLE
update fpga team

### PR DESCRIPTION
**Pull request by namachan10777**
FPGAチームのメンバーを2024年現在のものに更新します。
Alumni情報のあたりは多分正確性を欠いています。

### CODE REVIEW

File: src/content/member/furukawa.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: 古川 和輝
 eng_name: Kazuki Furukawa
```
Comment (LIGHTFRANTICE) [inline src/content/member/furukawa.yml:2]: eng_name: Kazuki FURUKAWA

File: src/content/member/kuwazawa.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: 桑澤 玄
 eng_name: Gen Kuwazawa
```
Comment (LIGHTFRANTICE) [inline src/content/member/kuwazawa.yml:2]: eng_name: Gen KUWAZAWA

File: src/content/member/melon.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: 顧超渊
+name: 顧 超渊
 eng_name: Gu Chaoyuan
```
Comment (LIGHTFRANTICE) [inline src/content/member/melon.yml:2]: eng_name: GU CHAOYUAN

File: src/content/member/niwase.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name: 庭瀬 稜平
 eng_name: Ryohei Niwase
```
Comment (LIGHTFRANTICE) [inline src/content/member/niwase.yml:2]: eng_name: Ryohei NIWASE

File: src/content/member/yharada.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 name: 原田 雄大
 eng_name: Yudai Harada
```
Comment (LIGHTFRANTICE) [inline src/content/member/yharada.yml:2]: eng_name: Yudai HARADA

File: src/content/alumni/mnishimura.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+year: 2023
+month: 3
+name: masaru nishimura
```
Comment (LIGHTFRANTICE) [inline src/content/alumni/mnishimura.yml:4]: name: Masaru Nishimura

**Review by LIGHTFRANTICE — COMMENTED** [review]
So sorry for the late review.

The change to the current members seems OK to me.  The eng_names may need an accordance.

Maybe there are other alumni students too ? I will ask Prof. Yamaguchi later on the meeting. And i will contact to you later about this.

Sorry for the late review again.

Sincerely.
Gu Xiangyu

File: src/content/member/gu.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
-name: ""
+name: 顧 翔宇
 eng_name: GU XIANGYU
```
Comment (LIGHTFRANTICE) [inline src/content/member/gu.yml:2]: eng_name: Xiangyu GU

File: src/content/alumni/mnishimura.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+---
+year: 2023
```
Comment (LIGHTFRANTICE) [inline src/content/alumni/mnishimura.yml:2]: year: 2024

File: src/content/alumni/oshima.yml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+---
+name: Yoshiyuki Oshima
+month: 3
+year: 2023
```
Comment (LIGHTFRANTICE) [inline src/content/alumni/oshima.yml:4]: year: 2024

File: src/content/alumni/yoon.yml [hunk 10]
```diff
@@ -0,0 +1,4 @@
+name: YOON JONG KWAN
+year: 2023
```
Comment (LIGHTFRANTICE) [inline src/content/alumni/yoon.yml:2]: year: 2024

File: src/content/alumni/taoyining.yml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+---
+name: Tao Yining
+year: 2023
```
Comment (LIGHTFRANTICE) [inline src/content/alumni/taoyining.yml:3]: year: 2024

**Comment by namachan10777** [comment]
Thank you for your review. eng_name will be fixed today.

The deployment is planned to take place after checking about alumni.

**Comment by namachan10777** [comment]
I fixed.

File: src/content/member/meng.yml [hunk 12]
```diff
@@ -1,7 +1,7 @@
-name: ""
+name: 孟 暉
 eng_name: Meng Hui
```
Comment (LIGHTFRANTICE) [inline src/content/member/meng.yml:2]: eng_name: Hui Meng

**Pull request merged**